### PR TITLE
Use Ajv@7, remove dependency on Ajv

### PR DESCRIPTION
**Pull request by mcollina**
Moving to a pre-compiled schema load time of this library by 40%.

On master:

```
$ time node index

real    0m0.101s
user    0m0.082s
sys     0m0.021s
```

This branch:

```
$ time node index

real    0m0.059s
user    0m0.046s
sys     0m0.017s
```

Ref. https://github.com/fastify/fastify/issues/2742

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "test": "npm run lint && npm run unit && npm run tsd",
     "lint": "standard",
-    "unit": "tap test/test.js test/*.test.js --100",
+    "unit": "tap test/test.js test/*.test.js",
```
Comment (kibertoad) [inline package.json:27]: Why did the coverage go down?

Comment (mcollina) [reply package.json:27]: The `validate.js` file is auto-generated and it has a lower coverage threshold. Note that I just moved the `--100` option to `.taprc`.

**Review by jsumners — COMMENTED** [review]
If we are going to rely on generated code that has to be shipped with the library, then there should be a pre-commit, pre-package, or something that makes sure the file is up-to-date.

**Comment by mcollina** [comment]
@jsumners done

**Comment by jsumners** [comment]
For those who haven't used the "prepare" script before:

image

https://docs.npmjs.com/cli/v6/using-npm/scripts

**Review by jsumners — APPROVED** [review]
LGTM

**Review by delvedor — APPROVED** [review]
LGTM

**Review by salmanm — APPROVED** [review]

**Comment by Eomm** [comment]
There are some issues with this PR because it has been reverted?

**Comment by mcollina** [comment]
Yes, it has a runtime dependency on Ajv@7.

**Comment by jsumners** [comment]
🤣

**Pull request merged**